src/components/CreateButton.tsx

```diff
@@ -209,7 +286,7 @@ export const CreateButton = () => {
             id="create-swap-button"
             data-testid="create-swap-button"
             class={buttonClass()}
-            disabled={buttonDisable() || !online()}
+            disabled={buttonDisable()}
```

Comment:
![image](https://github.com/BoltzExchange/boltz-web-app/assets/8138838/2c9b9211-1f5a-4626-bb59-88b690b6828d)
Invoices of the wrong network have a yellow button that can't be clicked. How about setting the class of the button when the disabled status is changed?